iidm/iidm-mergingview/src/main/java/com/powsybl/iidm/mergingview/TieLineAdapter.java

```diff
@@ -34,4 +34,9 @@ public HalfLine getHalf1() {
     public HalfLine getHalf2() {
         return ((TieLine) getDelegate()).getHalf2();
     }
+
+    @Override
+    public HalfLine getHalf(Side side) {
+        return ((TieLine) getDelegate()).getHalf(side);
```

Comment:
I wonder if we should override getDelegate() to return a TieLine and avoid casting everywhere

Reply:
As it is an AbstractAdapter<Line>, I do not think it is possible to change return type of getDelegate to TieLine.